packages/manager/modules/vps/src/dashboard/tile/configuration/service.js

```diff
@@ -48,6 +48,16 @@ export default class ConfigurationTileService {
     );
   }
 
+  static getPlanPriceTotal(upperPlan) {
+    const upperPlanTotalPriceValue = ConfigurationTileService.getPlanPriceValue(
```

Comment:
Suggestion: Since upperPlanTotalPriceValue is populated twice in function getPlanPriceTotal() and getPlanUpFrontPriceTotal(), can we pre populate it once and call these function ?